### PR TITLE
Clarified where to make the test folder

### PR DESCRIPTION
**Pull request by meeple142**
## Summary

If you make the `test` folder in the `src` folder then the paths are wrong in the xml.

This is linked to #11866 
I think they updated the paths but not the directions.

### CODE REVIEW

File: docs/core/tutorials/testing-with-cli.md [hunk 1]
```diff
@@ -97,7 +97,25 @@ Optional exercise: You can add a new pet type, such as a `Bird`, by extending th
 
 The `NewTypes` project is in place, and you've organized it by keeping the pets-related types in a folder. Next, create your test project and start writing tests with the [xUnit](https://xunit.github.io/) test framework. Unit testing allows you to automatically check the behavior of your pet types to confirm that they're operating properly.
 
-Navigate back to the *src* folder and create a *test* folder with a *NewTypesTests* folder within it. At a command prompt from the *NewTypesTests* folder, execute `dotnet new xunit`. This produces two files: *NewTypesTests.csproj* and *UnitTest1.cs*.
+Navigate back to the *root* folder and create a *test* folder with a *NewTypesTests* folder within it.
```
Comment (tdykstra) [inline docs/core/tutorials/testing-with-cli.md]:
```suggestion
Navigate back to the *NewTypes* folder and create a *test* folder with a *NewTypesTests* folder within it.
```

**Review by tdykstra — COMMENTED** [review]
@meeple142 Thanks for this correction -- please click the CLA "sign now" link so we can merge this PR, and take a look at my suggested change.

**Comment by tdykstra** [comment]
@meeple142 please click the CLA "sign now" link so we can merge this PR.

**Comment by BillWagner** [comment]
closing due to lack of response.

@meeple142 If you want us to merge this, please re-open, sign the CLA, and respond to the review suggestions.

**Pull request closed without merging**